angular_bind_polymer.js

```diff
@@ -2,6 +2,7 @@ angular.module('eee-c.angularBindPolymer', []).
 directive('bindPolymer', function() {
   return {
     restrict: 'A',
+    scope: {}, /* Scope added here */
```

Comment:
Now I know angular bit more than what i knew at the time of submitting this request. Thanks for the valuable suggestions. I will try them for sure.